### PR TITLE
Update people.yaml

### PR DESCRIPTION
**Pull request by thessaly**
Hi there, adding my profile info for the experts' list.

### CODE REVIEW

File: _data/people.yaml [hunk 1]
```diff
@@ -1658,6 +1658,27 @@ sudarshangc:
...
+    email: jarancio@unsam.edu.ar
+    website: https://julieta.aranc.io/
+    orcid: 0000-0002-8624-4182
+    expertise:
+       - How to open your science hardware prototypes
+       - How to contribute/use available open science hardware in your research
+       - How to run effective and inclusive citizen/community science projects
+       - Using Wikidata and other linked data tools to boost your research
+       - General panorama of Open science in Latin America
+       - Thinking what 'science and knowledge democratization' means in context
```
Comment (malvikasharan) [inline _data/people.yaml:1674]:
```suggestion
       hardware prototypes, contributing and using open science hardware, inclusive citizen/community science projects, Wikidata and other linked data tools, Open science in Latin America, 'science and knowledge democratization' in context
```

Comment (malvikasharan) [reply _data/people.yaml:1674]: @thessaly is it ok to keep it as keyword-ish?

Comment (bebatut) [reply _data/people.yaml:1674]: @malvikasharan I quickly reformated your changes as a list of keywords for the rendering on the website. I hope it is ok

Comment (thessaly) [reply _data/people.yaml:1674]: sure, feel free to adapt!

**Comment by bebatut** [comment]
Thanks @thessaly for adding yourself 😄

**Pull request merged**